.github/workflows/update_latest_versions.yml

```diff
@@ -0,0 +1,52 @@
+name: Update Latest Versions
+
+on:
+  workflow_dispatch: # can be triggered manually
+  schedule:
+    - cron: "0 0 * * 0" # weekly on Sunday at midnight
+
+jobs:
+  update-latest-versions:
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+      actions: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
nitpicky but do you need to checkout the entire history?
`1` is the default and just grabs the latest